Makefile

```diff
@@ -118,13 +119,33 @@ $(BUILD_DIR)/$(PROJECT)-%: $(STATIK_FILES) $(GO_FILES) $(BUILD_DIR) deploy/cross
 	(cd `dirname $@`; shasum -a 256 `basename $@`) | tee $@.sha256
 	file $@ || true
 
+.PHONY: kokoro-cross
+kokoro-cross: $(foreach platform, $(SUPPORTED_PLATFORMS), $(KOKORO_BUILD_DIR)/$(PROJECT)-$(platform))
+
+$(KOKORO_BUILD_DIR)/$(PROJECT)-%: $(STATIK_FILES) $(GO_FILES) $(KOKORO_BUILD_DIR)
+	$(eval os = $(firstword $(subst -, ,$*)))
+	$(eval arch = $(lastword $(subst -, ,$(subst .exe,,$*))))
+	$(eval ldflags = $(GO_LDFLAGS) $(patsubst %,-extldflags \"%\",$(LDFLAGS_$(os))))
+	$(eval tags = $(GO_BUILD_TAGS_$(os)) $(GO_BUILD_TAGS_$(os)_$(arch)))
+	GOOS=$(os) GOARCH=$(arch) CGO_ENABLED=1 go build -tags "$(tags)" -ldflags "$(ldflags)" -o $@ ./cmd/skaffold
+	(cd `dirname $@`; shasum -a 256 `basename $@`) | tee $@.sha256
+
+.PHONY: upload-kokoro-release
+upload-kokoro-release: $(BUILD_DIR)/VERSION
```

Comment:
I was thinking that if we're replacing the release process to use kokoro, then we should *remove* the release stuff here so as to prevent someone from accidentally releasing outside of kokoro.  So maybe this copying process should be moved into hack/kokoro/?

Reply:
I'll move these into a makefile in the `hack/kokoro` directory, that way we can still take advantage of some of the handy makefile features